site/content/cluster/binaries_cluster.md

```diff
@@ -0,0 +1,260 @@
+---
+linktitle: "Binaries"
+weight: 2
+---
+
+# Creating an M3 Cluster with Binaries
+
+This guide shows you the steps involved in creating an M3 cluster using M3 binaries, typically you would automate this with infrastructure as code tools such as Terraform or [Kubernetes](/docs/operator).
+
+{{% notice note %}}
+This guide assumes you have read the [quickstart](/docs/quickstart), and builds upon the concepts in that guide.
+{{% /notice %}}
+
+## M3 Architecture
+
+Here's a typical M3 deployment:
+
+<!-- TODO: Update image -->
+
+![Typical Deployment](/cluster_architecture.png)
+
+An M3 deployment typically has two main node types:
+
+-   **Coordinator node**: `m3coordinator` nodes coordinate reads and writes across all nodes in the cluster. It's a lightweight process, and does not store any data. This role typically runs alongside a Prometheus instance, or is part of a collector agent such as statsD.
+-   **Storage node**: The `m3dbnode` processes are the workhorses of M3, they store data and serve reads and writes.
+
+And exposes two ports:
+
+-   `7201` to manage the cluster topology, you make most API calls to this endpoint
+-   `7203` for Prometheus to scrape the metrics produced by M3DB and M3Coordinator
+
+## Prerequisites
+
+M3 uses [etcd](https://etcd.io/) as a distributed key-value storage for the following functions:
+
+-   Update cluster configuration in realtime
+-   Manage placements for distributed and sharded clusters
+
+## Download and Install a Binary
+
+You can download the latest release as [pre-compiled binaries from the M3 GitHub page](https://github.com/m3db/m3/releases/latest). Inside the expanded archive are binaries for `m3dbnode`, which combines a coordinator and storage node, and a binary for `m3coordinator`, which is a standalone coordinator node.
+
+## Build from Source
+
+### Prerequisites
+
+-   [Go 1.10 or higher](https://golang.org/dl/)
```

Comment:
Should we say like recommended to use `1.15` or something? @robskillington

Reply:
Let's just remove the version here, just say "Go".